lib/ts/recipe/emailpassword/components/themes/resetPasswordUsingToken/enterEmail.tsx

```diff
@@ -23,8 +23,9 @@ import StyleContext from "../../../../../styles/styleContext";
 import { EnterEmailThemeProps, EnterEmailThemeState } from "../../../types";
 
 import FormBase from "../../library/formBase";
+import { withOverride } from "../../../../../components/componentOverride/withOverride";
 
-export default class EnterEmailTheme extends PureComponent<EnterEmailThemeProps, EnterEmailThemeState> {
+class EnterEmailTheme extends PureComponent<EnterEmailThemeProps, EnterEmailThemeState> {
```

Comment:
Shouldn't the name of this component be `EmailPasswordEnterEmailTheme`? So that it shows up as that in the component tree?

Reply:
Addressed in https://github.com/supertokens/supertokens-auth-react/pull/255/commits/f604b0b28f247d45345d984f714c68171f284650

The explanation for the approach is in the commit message. TL;DR:
- named component variables the same as overrides, so that it's consistent with React inspector
- made named exports matching the name of the file (reducing the number of changes in consuming files)

Reply:
Can it be resolved?